version.mk

```diff
@@ -1,3 +1,3 @@
 vsn_major=3
-vsn_minor=4
-vsn_patch=2
+vsn_minor=5
+vsn_patch=0
```

Comment:
Usually we bump this right before the release tagging. But this is fine as well. Main will be 3.5.0 we already decided.

Reply:
I had to do this so the test build showed my new page :)

Reply:
That makes sense!